samples/function_chaining_custom_status/DurableOrchestration/__init__.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+import azure.functions as func
+import azure.durable_functions as df
+
+
+def orchestrator_function(context: df.DurableOrchestrationContext):
+    """This function provides the core function chaining orchestration logic

```

Comment:
The comments here refer to `function chaining` which we're doing, but I would want them to also highlight that they are setting custom statuses, which I believe the user can see if they click on the returned 'status URL'....just to differentiate this sample's comments from `function_chaining`